mp4parse/tests/public.rs

```diff
@@ -594,3 +599,42 @@ fn public_video_av1() {
         }
     }
 }
+
+#[test]
+fn public_avif_primary_item() {
+    let context = &mut mp4::AvifContext::new();
+    let input = &mut File::open(IMAGE_AVIF).expect("Unknown file");
+    mp4::read_avif(input, context).expect("read_avif failed");
+    assert_eq!(context.primary_item.len(), 6979);
+    assert_eq!(context.primary_item[0..4], [0x12, 0x00, 0x0a, 0x0a]);
+}
+
+#[test]
+#[ignore] // Remove when we add support
+fn public_avif_primary_item_is_grid() {
+    let context = &mut mp4::AvifContext::new();
+    let input = &mut File::open(IMAGE_AVIF_GRID).expect("Unknown file");
+    mp4::read_avif(input, context).expect("read_avif failed");
+    // Add some additional checks
```

Comment:
Re comment, do we want any additional checks before merging this, or will we handle this case when we un-ignore the test?

Reply:
I think the nature of the additional checks will be more evident once we add the support for non-av01 primary items. I've added https://github.com/mozilla/mp4parse-rust/issues/198 to track that, and will add a link to the code here.